app/graphql/resolvers/webhook_resolver.rb

```diff
@@ -0,0 +1,25 @@
+# frozen_string_literal: true
+
+module Resolvers
+  class WebhookResolver < Resolvers::BaseResolver
+    include AuthenticableApiUser
+    include RequiredOrganization
+
+    REQUIRED_PERMISSION = "developers:manage"
+
+    description "Query a webhook"
+
+    argument :id, ID, required: true
+
+    type Types::Webhooks::Object, null: true
+
+    def resolve(id:)
+      webhook = current_organization.webhook_endpoints.joins(:webhooks)
+        .find_by(webhooks: {id:})&.webhooks&.find(id)
```

Comment:
I'm so confused by this query. The `find_by.webhooks.find` is brain fucking me.

I was expecthing this:
```suggestion
      webhook = current_organization.webhooks.find_by(id:)
```

Reply:
I have the same question

Reply:
good spot thanks